lib/dradis/plugins/thor_helper.rb

```diff
@@ -5,7 +5,7 @@ module ThorHelper
       attr_accessor :task_options, :logger
 
       def detect_and_set_project_scope
-        ;
+        task_options[:project_id] = Project.new.id
```

Comment:
@aapomm this one will work in CE (it will be 1), but not so sure in Pro ?

Reply:
In Pro, this method is overridden in thor_helper which sets the right project.